controllers/drplacementcontrolvolsync.go

```diff
@@ -329,7 +329,7 @@ func (d *DRPCInstance) ResetVolSyncRDOnPrimary(clusterName string) error {
 		return nil
 	}
 
-	mw, err := d.mwu.FindManifestWork(rmnutil.MWTypeVRG, clusterName)
+	mw, err := d.mwu.FindManifestWorkByType(rmnutil.MWTypeVRG, clusterName)
```

Comment:
ok found the related issued in the bug, seem to be this:
https://github.com/RamenDR/ramen/pull/892#discussion_r1308803823

Since only this change is needed to fix the issue, lets do only this change and do the logging changes later.

Reply:
The logging is important for this fix. It took forever to figure out the problem.